middleware/middleware-1.3.11.md

```diff
@@ -0,0 +1,63 @@
+# fiskaltrust.Middleware 1.3.11 (Germany)
+_October 28, 2020_
+
+With this version, we're happy to announce that the fiskaltrust Android Middleware is now publicly available in production! Additionally, we added support for the newly available Swissbit Cloud TSE, and introduced several important stability and performance improvements to our SCUs.
+
+<div class="alert alert-warning" role="alert" style="border-radius: 0">Version 1.3 of the Middleware is meant for the German market only, customers in Austria and France should continue to use version 1.2. We will unify these experiences in an upcoming version.</div>
+
+## New SCU: SwissbitCloud (Sandbox)
+To increase our portfolio of supported cloud TSEs, we added a new SCU package that enables our Middleware users to include the newly available Swissbit Cloud TSE. 
+
+Like the _DeutscheFiskal_ SCU, this new package automatically downloads and initializes the Fiscal Cloud Connector (the backend web service that is required to operate this TSE). If required, it's also possible to bypass this functionality and manually install this service by passing the `FccDirectory` parameter.
+
+While this TSE is running smoothly in our tests so far, there are some limitations, both technically and due to the vendor's business model:
+- The SwissbitCloud TSE was only certified for Windows 10 and Ubuntu 20.04 so far. Although the producer is working on additional certifications for other OSs, it can meanwhile officially only be operated on these systems. We're in continuous communication with Swissbit and will notify our users as soon as there are only news about this topic.
+- Due to the pricing model (pricing is calculated per client, not per TSE), our usual rollout scenarios are limited. We are working on updating our [documentation](https://docs.fiskaltrust.cloud/doc/productdescription-de-doc/for-posdealers/02-pre-sales/rollout-scenarios.html) to reflect these limitations.
```

Comment:
Should we mention sth about the performance?

Reply:
Good point, thanks - added a paragraph